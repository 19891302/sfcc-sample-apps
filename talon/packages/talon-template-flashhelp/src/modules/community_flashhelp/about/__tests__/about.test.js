import { createElement } from 'lwc';
import About from 'community_flashhelp/about';

describe('community_flashhelp/about', () => {
    beforeEach(() => {
        while (document.body.firstChild) {
            document.body.removeChild(document.body.firstChild);
        }
    });

    it('renders correctly', () => {
        const element = createElement('community_flashhelp-about', { is: About });
        document.body.appendChild(element);

        return Promise.resolve().then(() => {
            expect(element).toMatchSnapshot();
        });
    });
});
