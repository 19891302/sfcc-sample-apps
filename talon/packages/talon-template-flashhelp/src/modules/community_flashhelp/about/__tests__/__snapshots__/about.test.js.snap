// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`community_flashhelp/about renders correctly 1`] = `
<community_flashhelp-about>
  #shadow-root(open)
    <div
      class="slds-align_absolute-center slds-text-heading_large"
      style="height: 5rem;"
    >
      About
    </div>
</community_flashhelp-about>
`;
